webapp/components/gh-login.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2018 The WPT Dashboard Project. All rights reserved.
```

Comment:
Nit: let's make it 2019 :)

Also nit: I don't think it's worthwhile saving four characters here. Let's call this file and the component `github-login`.